src/components/IssueListItem.js

```diff
@@ -7,6 +7,10 @@ import {IssueOpenedIcon, CommentIcon, MilestoneIcon} from "@primer/octicons-reac
 
 dayjs.extend(relativeTime);
 
+function fontContrast(color) {
+  return Number("0x" + color) > 16000000 ? "black" : "white";
+}
```

Comment:
@filiptronicek Moving this to `lib` folder would be nice! This helps us to reuse this function everywhere!